src/main/java/net/dries007/tfc/types/DefaultRecipes.java

```diff
@@ -390,10 +390,8 @@ public static void onRegisterHeatRecipeEvent(RegistryEvent.Register<HeatRecipe>
             new HeatRecipeSimple(IIngredient.of(ItemFoodTFC.get(Food.VENISON)), new ItemStack(ItemFoodTFC.get(Food.COOKED_VENISON)), 200, 480).setRegistryName("cooked_venison"),
             new HeatRecipeSimple(IIngredient.of(ItemFoodTFC.get(Food.RABBIT)), new ItemStack(ItemFoodTFC.get(Food.COOKED_RABBIT)), 200, 480).setRegistryName("cooked_rabbit"),
             new HeatRecipeSimple(IIngredient.of(ItemFoodTFC.get(Food.WOLF)), new ItemStack(ItemFoodTFC.get(Food.COOKED_WOLF)), 200, 480).setRegistryName("cooked_wolf"),
-            new HeatRecipeSimple(IIngredient.of(ItemFoodTFC.get(Food.CHEVON)), new ItemStack(ItemFoodTFC.get(Food.COOKED_CHEVON)), 200, 480).setRegistryName("cooked_chevon"),
```

Comment:
So, there's no more chevon, duck and mongoose foods? Did you remove them from the lootables too? This is more of a concern for backwards compatibility, but I think that it will only delete them across inventories.

Reply:
changed on purpose, with all the animals being added it would have created a lot of new textures, so the meats were condensed to help with this. see the animal chart pinned in the discord chat for reference of the changes. the Lang file was changed to reflect how it should be done, but the items were kept the same to help with backwards compatibility. livestock birds all drop Poultry(chicken meat/texture) all huntable birds now drop Fowl( pheasent meat/texture) as an example

Reply:
the mongoose, re uses the hyena texture, and all dog/canine now drop the wolf meat/texture